developer-extension/src/common/types.ts

```diff
@@ -12,13 +12,13 @@ export interface BackgroundActions {
 
 export interface PopupActions {
   newStore: Store
+  newEvents: StoredEvent[]
 }
 
 export interface Store {
   devServerStatus: 'unavailable' | 'checking' | 'available'
   useDevBundles: boolean
   useRumSlim: boolean
-  logEventsFromRequests: boolean
   blockIntakeRequests: boolean
   local: {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏Since the local store is not used anymore after your two PRs, could you remove it?

Reply:
Yes, this is the idea ;)